### PR TITLE
applications: nrf_desktop: Add support for nRF53 DK working as dongle

### PR DESCRIPTION
**Pull request by MarekPieta**
Change introduces support for nRF53 DK working as nRF Desktop dongle.

Jira: NCSDK-10094

### CODE REVIEW

**Review by pdunaj — APPROVED** [review]

File: applications/nrf_desktop/configuration/nrf5340dk_nrf5340_cpuapp/ble_scan_def.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+/* This structure enforces the header file is included only once in the build.
+ * Violating this requirement triggers a multiple definition error at link time.
+ */
+const struct {} ble_scan_include_once;
+
+#include "ble_event.h"
+
+static const char * const peer_name[] = {
+	[PEER_TYPE_MOUSE] = "Mouse nRF52 Desktop",
+	[PEER_TYPE_KEYBOARD] = "Keyboard nRF52 Desktop",
```
Comment (emob-nordic) [inline applications/nrf_desktop/configuration/nrf5340dk_nrf5340_cpuapp/ble_scan_def.h:20]: nRF52 -> nRF53 ?

**Review by emob-nordic — CHANGES_REQUESTED** [review]
Just this one little thing.

File: applications/nrf_desktop/configuration/nrf5340dk_nrf5340_cpuapp/ble_scan_def.h (hunk 1 above)
Comment (MarekPieta) [reply applications/nrf_desktop/configuration/nrf5340dk_nrf5340_cpuapp/ble_scan_def.h:20]: These are used as Bluetooth device names by nRF Desktop peripherals. We need to use the same names here to let the dongle connect with the peripherals.

Comment (MarekPieta) [reply applications/nrf_desktop/configuration/nrf5340dk_nrf5340_cpuapp/ble_scan_def.h:20]: We could consider changing this as a separate PR, but I am not sure if the change is actually needed.

Comment (emob-nordic) [reply applications/nrf_desktop/configuration/nrf5340dk_nrf5340_cpuapp/ble_scan_def.h:20]: Okay. I understand the reasoning for that. Maybe in the future.

**Review by emob-nordic — APPROVED** [review]

**Pull request merged**